tensorflow/examples/android/src/org/tensorflow/demo/StylizeActivity.java

```diff
@@ -484,114 +470,41 @@ private void resetPreviewBuffers() {
     initializedSize = desiredSize;
   }
 
-  /*
-   * Callback for android.hardware.Camera API
-   */
-  @Override
-  public void onPreviewFrame(final byte[] bytes, final Camera camera) {
-    if (computing) {
-      return;
-    }
-    computing = true;
+  protected void processImageRGBbytes(byte[] luminance) {
+    if (desiredSize != initializedSize) {
+      LOGGER.i(
+              "Initializing at size preview size %dx%d, stylize size %d",
+              previewWidth, previewHeight, desiredSize);
+      rgbFrameBitmap = Bitmap.createBitmap(previewWidth, previewHeight, Config.ARGB_8888);
+      croppedBitmap = Bitmap.createBitmap(desiredSize, desiredSize, Config.ARGB_8888);
 
-    Camera.Size previewSize = camera.getParameters().getPreviewSize();
-    try {
-      // Initialize the storage bitmaps once when the resolution is known.
-      if (previewWidth != previewSize.width || previewHeight != previewSize.height) {
-        onPreviewSizeChosen(new Size(previewSize.width, previewSize.height), 90);
-      }
-      if (desiredSize != initializedSize) {
-        resetPreviewBuffers();
-      }
+      frameToCropTransform =
```

Comment:
4 space indentation on continuation of statement until sequence point (we don't use 8 space indentations anywhere)